alphashield/database/mongodb_client.py

```
@@ -1,132 +1,73 @@
+                            "agent_id": decision["agent_id"],
+                            "loan_id": decision["loan_id"],
+                            "timestamp": decision["timestamp"],
+                        },
+                        {"$setOnInsert": decision},
+                        upsert=True,
+                    )
+                except Exception as e:
+                    raise ExecutionError(f"mongo store failed: {e}")
+
+        return _RealClient()
+    except Exception:
```

Ruff (0.14.1)

Do not catch blind exception: `Exception`

(BLE001)